lib/resty/acme/dnsapi/dynv6.lua

```diff
@@ -0,0 +1,160 @@
+local http = require("resty.http")
+local cjson = require("cjson")
+
+local _M = {}
+local mt = {__index = _M}
+
+function _M.new(token)
+  if not token or token == "" then
+    return nil, "api token is needed"
+  end
+
+  local self = setmetatable({
+    endpoint = "https://dynv6.com/api/v2",
+    httpc = nil,
+    token = token,
+    zone = nil,
+    headers = {
+      ["Content-Type"] = "application/json",
+      ["Authorization"] = "Bearer " .. token,
+    }
+  }, mt)
+
+  self.httpc = http.new()
+  return self
+end
+
+function _M:get_zone_id(fqdn)
```

Comment:
for "non-public" functions, consider write it in form of 
```lua
local function get_zone_id(self, fqdn)
```
So that it can't be used outside of this module. This will help us keep the API interface for dns providers clean.